bids/layout/index.py

```diff
@@ -164,10 +165,21 @@ def index_files(self):
         """Index all files in the BIDS dataset. """
         self._index_dir(self.root, self.config)
 
-    def index_metadata(self):
+    def index_metadata(self, **filters):
         """Index metadata for all files in the BIDS dataset. """
```

Comment:
Docstring should mention the `filters` argument explicitly.

Reply:
thanks for pointing this out, added one in.